src/scripts/main.js

```diff
@@ -1,3 +1,76 @@
 'use strict';
 
-// write code here
+const employees = document.querySelectorAll('li');
```

Comment:
так будет стразу массив

```suggestion
const initialList = document.querySelector('ul');
const employees = [...initialList.children];
```

Reply:
Какой результат сортировки ожидается, я видел в задании, но не до конца понимал, должен ли сортироваться тот же список или создаваться новый рядом с тем, что уже есть :) Теперь понял.

const employees = [...initialList.children]; – получается, на коллекции можно спред-оператор использовать для получения из нее массива, как и на объекте. В принципе, это логично, но сам бы не додумался, наверное. Спасибо)

Единственный вопрос - а разве мы в sortList не коллекцию должны передать?

Reply:
И еще я сначала подумал, что getEmployees нужно использовать внутри sortList. Но судя по всему эта функция совсем для других целей нужна - просто чтобы получать массив объектов (вроде как получаем своеобразную базу данных, которую потом можем где-то использовать). Правильно?